composer.json

```diff
@@ -18,7 +18,7 @@
     "matthiasnoback/symfony-config-test": "^4.3",
     "phpdocumentor/reflection-docblock": "^5.2",
     "phpunit/php-token-stream": "^3.1.3|^4.0.4",
-    "phpunit/phpunit": "^8.5|^9.0",
+    "phpunit/phpunit": "^9.6|^10.4",
     "sebastian/phpcpd": "^4.1|^5.0|^6.0",
     "squizlabs/php_codesniffer": "^3.6",
     "malukenho/docheader": "^0.1.8",
```

Comment:
I think this can be changed to "malukenho/docheader": "^0.1.8|^1.0",
But needs check for BC breaks.

This will fix the failing composer install. https://github.com/OpenConext/Monitor-bundle/actions/runs/6704694822

Reply:
Thanks for the suggestion @Chris53897 

I'll be updating and testing this right away